include/maths/CBayesianOptimisation.h

```diff
@@ -8,6 +8,11 @@
 #include <maths/CPRNG.h>
 #include <maths/ImportExport.h>
 
+#include <core/CDataSearcher.h>
+#include <core/CStatePersistInserter.h>
+#include <core/CStateRestoreTraverser.h>
+#include <core/CoreTypes.h>
```

Comment:
nit: as with other code can we keep includes in library dependency order, i.e. core first.

Reply:
Done